File: app/src/main/java/com/robotnec/budget/app/util/ViewUtil.java
package com.robotnec.budget.app.util;

import android.content.Context;
import android.content.res.TypedArray;

import com.robotnec.budget.R;

/**
 * @author zak <zak@swingpulse.com>
 */
public class ViewUtil {

    public static int getToolbarHeight(Context context) {
        final TypedArray styledAttributes = context.getTheme().obtainStyledAttributes(
                new int[]{R.attr.actionBarSize});
        int toolbarHeight = (int) styledAttributes.getDimension(0, 0);
        styledAttributes.recycle();

        return toolbarHeight;
    }
}

File: app/src/main/java/com/robotnec/budget/app/util/ViewUtil.kt
package com.robotnec.budget.app.util

import android.content.Context
import com.robotnec.budget.R

/**
 * @author zak zak@swingpulse.com>
 */
object ViewUtil {

    fun getToolbarHeight(context: Context): Int {
        val styledAttributes = context.theme.obtainStyledAttributes(
                intArrayOf(R.attr.actionBarSize))
        val toolbarHeight = styledAttributes.getDimension(0, 0f).toInt()
        styledAttributes.recycle()

        return toolbarHeight
    }
}
